kythe-verification/testdata/basic/ImportsRef.hs

```diff
@@ -0,0 +1,8 @@
+-- Verify the references to the imported modules
```

Comment:
I like the idea. Resolved the TODO from CrossRef2 test.
Not sure if a negative test is useful. It's more like verifying Kythe framework than haskell-indexer logic.